tests/ngff/test_ngff.py

```diff
@@ -645,6 +646,70 @@ def test_set_scale(ch_shape_dtype):
             assert dataset.zattrs["iohub"]["prior_z_scale"] == 3.0
 
 
+@given(
+    ch_shape_dtype=_channels_and_random_5d_shape_and_dtype(),
```

Comment:
Why does this need to be parametrized?

Reply:
i guess we only need channel names. removing the other things.